### DISCUSSION TITLE
Welcome to afk_monorepo Discussions!

### DISCUSSION THREAD

**Discussion by MSghais** [AFK-AlignedFamKernel/afk_monorepo#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sravya-212** [comment]
@MSghais I have commented on an issue #72, I was not assigned but this is the message that I received "Hey @Sravya-212!
Thanks for showing interest.
We've created an application for you to contribute to AFK - Aligned Fam Kernel.
Go check it out on [OnlyDust](https://app.onlydust.com/p/afk---aligned-fam-kernel)!"
Can you tell me where I can access the code?
Cause I don't see any application or source
